### ISSUE TITLE
suggestion: Build-in prompts

### ISSUE THREAD

**Issue by urmi1510799** [chatgpt-web-dev/chatgpt-web#39]
is it possible to add build-in prompts?

For example:

Ready to use prompts after deploying. Like some essential prompts: paraphrasing, dictionary etc.. users don't have to import those prompts. these prompts is globally enabled for all users. I think this will make the project more attractive.

**Comment by Kerwin1202** [comment]
Yes, I am already working on this feature and have completed half of it. Due to work, it will probably be launched over the weekend.

**Comment by Kerwin1202** [comment]
The first step already supports custom fixed prompts for each room, and the next step will consider building in some prompts ，， docker 2.12.3

<img width="851" alt="image" src="https://user-images.githubusercontent.com/16319833/232826637-d140917c-3330-4681-8df1-5d2a0bc6d6ff.png">
